test-toolkit/docker/1.4.0/Dockerfile.pytorch

```diff
@@ -0,0 +1,41 @@
+from pytorch/pytorch:1.4-cuda10.1-cudnn7-runtime
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+ARG MMS_VERSION=1.0.8
+
+ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
+ENV TEMP=/home/model-server/tmp
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    libgl1-mesa-glx \
+    libglib2.0-0 \
+    libsm6 \
+    libxext6 \
+    libxrender-dev \
+    openjdk-8-jdk-headless \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN conda install -c conda-forge opencv==4.0.1 \
+    && ln -s /opt/conda/bin/pip /usr/local/bin/pip3
+
+RUN pip install mxnet-model-server==$MMS_VERSION
```

Comment:
should we use [`multi-model-server`](https://pypi.org/project/multi-model-server) instead? (it's newer)

Reply:
yeah, i was confused about it too - i remember us recently switching to it.
But this is what we use in the latest images:
https://github.com/aws/sagemaker-pytorch-serving-container/blob/master/docker/1.4.0/py3/Dockerfile.cpu#L68

Reply:
🤔 makes sense to match that